IFC4x3/Properties/p/ProcurementMethod_0HZB4plKf1oOsbtTq1O1Zs/Documentation.md

```diff
@@ -0,0 +1 @@
+The method by which an IfcProductType/IfcProduct is acquired and installed. The value provided shall be one of the following four character acronyms: “CFCI” (meaning Contractor Furnished Contractor Installed), “OFCI” (meaning Owner Furnished Contractor Installed), or “OFOI” (meaning Owner Furnished Owner Installed).
```

Comment:
PEnum?

Reply:
Says IfcLabel in the official documentation.